UsersApp/View Controllers/UserDetailsViewController.swift

```diff
@@ -0,0 +1,213 @@
+import UIKit
+
+
+class UserDetailsViewController: UIViewController, UITextViewDelegate {
+    
+    @IBOutlet weak var userImageView: UIImageView!
+    
+    @IBOutlet weak var noteTextView: UITextView!
+    
+    @IBOutlet weak var firstNameView: UserDetailView!
+    @IBOutlet weak var lastNameView: UserDetailView!
+    @IBOutlet weak var genderView: UserDetailView!
+    
+    @IBOutlet weak var streetAdressLabel: UILabel!
+    @IBOutlet weak var cityLabel: UILabel!
+    @IBOutlet weak var stateLabel: UILabel!
+    @IBOutlet weak var countryLabel: UILabel!
+    @IBOutlet weak var postalCodeLabel: UILabel!
+    @IBOutlet weak var coordinatesLabel: UILabel!
+    @IBOutlet weak var timezoneLabel: UILabel!
+    
+    @IBOutlet weak var emailView: UserDetailView!
+    @IBOutlet weak var dateOfBirth: UserDetailView!
+    @IBOutlet weak var registeredDateView: UserDetailView!
+    @IBOutlet weak var phoneView: UserDetailView!
+    @IBOutlet weak var cellphoneView: UserDetailView!
+    
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var saveButton: UIButton!
+    
```

Comment:
all outlets which is not used outside of controller class should be private. Available and for UsersController